flagging_site/config.py

```diff
@@ -140,6 +140,8 @@ class DevelopmentConfig(Config):
     DEBUG: bool = True
     TESTING: bool = True
     OFFLINE_MODE = strtobool(os.getenv('OFFLINE_MODE', 'false'))
+    BASIC_AUTH_USERNAME: str = 'admin'
+    BASIC_AUTH_PASSWORD: str = 'password'
```

Comment:
For the base config this is fine-- we can use this behavior in both development and testing.

For the ProductionConfig, can you override the value as `BASIC_AUTH_PASSWORD: str = os.environ['BASIC_AUTH_PASSWORD']` ? <-- What this does is forces the user to define it in the environment. I explicitly suggest `os.environ[key]` over either `os.environ.get(key)` or `os.getenv(key)` because I _want_ it to fail if the user who is deploying does not specify a password.

Reply:
Oh actually, I'd go a step further and also make the base config one `BASIC_AUTH_PASSWORD: str = os.getenv('BASIC_AUTH_PASSWORD', 'password')`. This gives someone the ability to test that setting a password with the env variables works as expected.